pkg/repoowners/repoowners.go

```diff
@@ -671,6 +682,46 @@ func (o *RepoOwners) entriesForFile(path string, people map[string]map[*regexp.R
 	return out
 }
 
+// MinimumReviewersForFile returns the minimum number of reviewers required to approve the requested file.
+// This is determined by the OWNERS file closest to the requested file.
+// If pkg/OWNERS has 2 minimum reviewers and pkg/util/OWNERS has 3 minimum reviewers this will return 3 for the path pkg/util/sets/file.go
+// If no minimum reviewers can be found then 0 is returned.
+func (o *RepoOwners) MinimumReviewersForFile(path string) int {
+	d := path
+	if !o.enableMDYAML || !strings.HasSuffix(path, ".md") {
+		// if path is a directory, this will remove the leaf directory, and returns "." for topmost dir
+		d = filepath.Dir(d)
+		d = canonicalize(d)
+	}
+
+	var foundMinReviewer *int
+	for {
+		relative, err := filepath.Rel(d, path)
+		if err != nil {
+			o.log.WithError(err).WithField("path", path).Errorf("Unable to find relative path between %q and path.", d)
+			foundMinReviewer = nil
+			break
+		}
+		for re, s := range o.minimumReviewers[d] {
+			if re == nil || re.MatchString(relative) {
+				foundMinReviewer = &s
+				break
+			}
+		}
+		if foundMinReviewer != nil || d == baseDirConvention {
+			// If we found a minimum reviewer count, or we've reached the base directory, break
+			break
+		}
+		d = filepath.Dir(d)
+		d = canonicalize(d)
+	}
+	if foundMinReviewer == nil {
+		// If we didn't find a minimum reviewer count, default to 0
+		return 0
```

Comment:
Should this be 1 to match current behaviour?
```suggestion
		return 1
```